src/town_gui.cpp

```diff
@@ -1177,6 +1187,20 @@ struct FoundTownWindow : Window {
 		this->UpdateButtons(true);
 	}
 
+	void OnInit() override
+	{
+		this->GetWidget<NWidgetStacked>(WID_TF_TOWN_ACTION_SEL)->SetDisplayedPlane(0);
+		this->GetWidget<NWidgetStacked>(WID_TF_ROAD_LAYOUT_SEL)->SetDisplayedPlane(0);
```

Comment:
Is there a reason to always set these, instead of having an `else` for the editor check below?

Reply:
In fact because it's not possible to switch game modes while leaving this window open, that wasn't necessary at all.